contrib/submit-simple-job/README.md

```diff
@@ -0,0 +1,50 @@
+Submit Simple Job
```

Comment:
two overall comments on document,

1. It needs overall context for the plugin. For example, who need that? what's value of it?
2. I have a look on UI, but it looks combine a lot of things together, and no instruction. It's not easy to understand features. So it's better to have instructions icon in UI to example each feature. If some feature/concept is from DLworkspace, it can be a reference.

Reply:
The updated doc could be read on https://github.com/Microsoft/pai/blob/qixcheng/webportal-plugin/submit-simple-job/contrib/submit-simple-job/README.md 

There is a bug in GitHub that the image link to the main branch.